source/features/mark-unread.tsx

```diff
@@ -270,7 +270,7 @@ function isCurrentSingleRepoPage({repository}: Notification): boolean {
 }
 
 function isParticipatingPage(): boolean {
-	return /\/notifications\/participating/.test(location.pathname);
+	return location.pathname.includes('/notifications/participating');
```

Comment:
Probably

```suggestion
	return location.pathname.startsWith('/notifications/participating');
```

Reply:
Yup, it's `https://github.com/notifications/participating`.